tool_list.yaml

```diff
@@ -3156,3 +3156,10 @@ tools:
 - name: rfove
   owner: imgteam
   tool_panel_section_label: Imaging
+- name: background_removal
+  owner: imgteam
+  tool_panel_section_label: Imaging
+- name: suite_omero_py
```

Comment:
Oddly, here we need to add the individual tools :(

Reply:
Done!